tiredrive/strategies.py

```diff
@@ -137,30 +168,32 @@ def maintain_winch(self):
     def auto_drive_until_liftable(self):
         robot = self.robot
         revs0 = robot.right_encoder.get()
-        while robot.right_encoder.get() <= revs0 + 60:
-            robot.forward(0.5)
+        while not robot.right_claw_whisker() and not robot.left_claw_whisker():
+            robot.forward(0.7)
             yield
 
     def drop_tote(self, i):
         robot = self.robot
         robot.winch_setpoint = robot.winch_setpoint_zero
+        durp = False
```

Comment:
no, its just a flag to ensure the if statement three lines down is only run once.